src/7-to-8/major-changes/platforms.rst

```diff
@@ -72,7 +72,7 @@ Platforms also define how hosts are selected from each platform:
 
 There may be cases where sets of platforms (for example a group of
 standalone compute servers, or a pair of mirrored HPC's) might be equally
-suitable for a task, but not share files systems to allow them to constitute
+suitable for a task, but not share file systems to allow them to constitute
```

Comment:
Whether they share file systems or not isn't really relevant.
I suggest ending the sentence after "equally suitable for a task".